src/DecisionProgramming.jl

```diff
@@ -30,7 +30,11 @@ export Node,
     InfluenceDiagram,
     generate_arcs!,
     generate_diagram!,
-    index_of,
+    indices,
+    I_j_indices,
+    indices_in_vector,
```

Comment:
Is this function in this pull request? I cannot find it from the changes on this page.

Reply:
Nevermind, now found it :D